tuning/sft_trainer.py

```diff
@@ -461,12 +461,46 @@ def parse_arguments(parser, json_config=None):
     )
 
 
-def main(**kwargs):  # pylint: disable=unused-argument
-    logger = logging.get_logger("__main__")
+def set_log_level(parsed_training_args):
+    """Set log level of python native logger and TF logger via argument from CLI or env variable.
+
+    Args:
+        parsed_training_args
+            Training arguments for training model.
+    """
+
+    # Clear any existing handlers if necessary
+    for handler in logging.root.handlers[:]:
+        logging.root.removeHandler(handler)
+
+    # Configure Python native logger log level
+    # If CLI arg is passed, assign same log level to python native logger
+    if parsed_training_args.log_level != "passive":
+        logging.basicConfig(level=parsed_training_args.log_level.upper())
+    else:
+        # Assign value of either env var LOG_LEVEL or warning
+        LOGLEVEL = os.environ.get("LOG_LEVEL", "WARNING").upper()
```

Comment:
Here if LOG_LEVEL is not set, it will default to logging.WARNING, which is an integer constant from the logging module (default is 30). The use of .upper() here will cause an error because .upper() is a string method and cannot be called on an integer.